shared/src/commonMain/kotlin/com/bunbeauty/shared/data/repository/OrderRepository.kt

```diff
@@ -96,18 +95,33 @@ class OrderRepository(
         token: String,
         userUuid: String,
     ): LightOrder? {
-        return orderDao.getLastOrderByUserUuid(userUuid)?.let(orderMapper::toLightOrder)
-            ?: networkConnector.getOrderList(token = token, count = 1).getNullableResult(
-                onError = {
-                    null
-                },
-                onSuccess = { orderServerList ->
-                    orderServerList.results.firstOrNull()?.let { orderServer ->
-                        saveOrderLocally(orderServer)
-                        orderMapper.toLightOrder(orderServer)
+        return if (cacheLastLightOrder.isValid) {
+            cacheLastLightOrder.lastOrder ?: orderDao.getLastOrderByUserUuid(userUuid)
```

Comment:
if cache is valid then convert it to `LightOrder` 
remove `orderDao.getLastOrderByUserUuid(userUuid)`

Reply:
?